Declare dependencies so we don't unnecessarily rebuild when running "make install"
--- Makefile.orig	2018-10-29 12:31:47.000000000 -0400
+++ Makefile	2021-04-28 23:26:11.000000000 +0800
@@ -21,7 +21,7 @@
 TAGS:
 	find . -name "*.[mhMH]" -exec etags -o ./TAGS -a '{}' +
 
-install: | Deployment backup-old-iterm
+install:
 	cp -R build/Deployment/iTerm2.app $(APPS)
 
 Development:
